SU-HSDO/suhumsci
# SHS-5784: New Well Color for Collections: Front-end

# READY FOR REVIEW

## Summary
Update the styles and logic to make the new two fields `Background Color (field_bg_color)` and `Background Color Width (field_bg_color_width)` work in the FE

## Need Review By (Date)
09/04

## Urgency
medium

## Steps to Test
1. On [Traditional](https://hs-traditional-91a52pchzdktwenvhcennatqscbeqci3.tugboatqa.com/test-shs-5784) and [Colorful](https://hs-colorful-91a52pchzdktwenvhcennatqscbeqci3.tugboatqa.com/test-5784)
2. Create a `Flexible Page` and add `Collections` for each `Background Color` and each `Background Color Width`
3. Verify the width and the color is correct in all different color pairings. It should look like what it's proposed in the design ([Traditional](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1532-5054&t=3gjoZR6wiJhj7ah4-4) / [Colorful](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1532-8341&t=3gjoZR6wiJhj7ah4-4)). The distribution of the colors in the design it's in the following way:

  - Color default value -> Colors in the first column (left side of the page)
  - Grey -> Color in the second column (center of the page)
  - Light -> Colors in the last column (right side of the page)

5. Please, also verify that existing Collection components that had a background set the well color looks as expected

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208267221345871
